src/views/login/security-question.html

```diff
@@ -33,39 +35,30 @@
 
             <!-- SECURITY ANSWER FIELDS -->
             <div ng-if="!security.loading && !security.invalidCode && security.isThereSelectedHospital()" class="container security-question--container">
-                <form name="formPassword" novalidate>
-                    <div class="form-group">
-                        <label name="text">{{security.Question}}</label>
-                        <div class="security-question--answer-box">
-                            <password-input>
-                                <input type="password"
-                                       id="security_answer_input"
-                                       autocapitalize="none"
-                                       autocorrect="off"
-                                       autocomplete="new-password"
-                                       name="text"
-                                       ng-model="security.answer"
-                                       ng-change="security.clearErrors()"
-                                       placeholder="{{'ENTERANSWERPLACEHOLDER'|translate}}"
-                                       style="touch-action: manipulation"/>
-                            </password-input>
-                        </div>
-                        <button class="button button--large shadow-2 button--large-green security-question--button"
-                                ng-disabled="security.submitting || security.tooManyAttempts"
-                                ng-click="security.submitAnswer(security.answer)">
-                            <span ng-if="!security.submitting">
-                                {{"CONTINUE"|translate}}
-                            </span>
-                            <span ng-if="security.submitting">
-                                <ons-icon icon="ion-load-d" spin="true"></ons-icon>
-                            </span>
-                        </button>
-
-                        <uib-alert class="security-question--alert" ng-show="security.alert.type && security.alertShow"
-                                   type="{{security.alert.type}}">
-                            {{security.alert.content|translate}}
-                        </uib-alert>
+                <form style="text-align: center" novalidate>
+                    <label for="securityAnswer">{{security.Question}}</label>
```

Comment:
[for](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/for) matches with `id`, so this should probably also be `security-answer-input`